Scripts/signin.js

```diff
@@ -0,0 +1,87 @@
+import * as enums from "./enums.js";
+import * as sign from "./sign.js";
+import * as regex from "./regex.js";
+
+const email = document.querySelector("#email");
+const pw = document.querySelector("#pw");
+const eyeIcon = document.querySelector(".eyeIcon");
+const form = document.querySelector("form");
+
+// Input focusout 시
+function formFocusOut(e) {
+  const targetName = e.target.name;
+  const targetValue = e.target.value;
+  // 이메일
+  if (targetName === "email") {
+    // 입력 값이 없을 때
+    if (targetValue === "") {
+      sign.makeWarningMsg(enums.warningMsg.EMAIL_NULL, e.target);
+      e.target.classList.add("warningForm");
+    }
+    // 올바른 형식이 아닐 때
+    else if (regex.emailFormat.test(targetValue) === false) {
+      sign.makeWarningMsg(enums.warningMsg.EMAIL_INVALID, e.target);
+      e.target.classList.add("warningForm");
+    }
+  }
+  // 비밀번호
+  else {
+    // 입력 값이 없을 때
+    if (targetValue === "") {
+      sign.makeWarningMsg(enums.warningMsg.PW_NULL, e.target);
+      e.target.classList.add("warningForm");
+    }
+  }
+}
+
+// Form 제출 시
+function submitForm(e) {
+  e.preventDefault();
+  email.dispatchEvent(new Event("focusin"));
+  pw.dispatchEvent(new Event("focusin"));
+  // 올바른 이메일, 비밀번호 입력했는지 확인
+  checkUser();
+}
+
+const checkUser = async function () {
+  try {
+    const response = await fetch(`${enums.urls.BASE_URL}/sign-in`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        email: `${email.value}`,
+        password: `${pw.value}`,
+      }),
+    });
+    if (!response.ok) {
+      throw new Error("존재하지 않는 계정입니다.");
+    }
+    const result = await response.json();
+    const signinToken = result.data.accessToken;
```

Comment:
이 코드는 아래와 같이 구조분해도 이용할 수 있습니다.

```suggestion
    const { data: { accessToken: signinToken } } = await response.json();
```